### PR TITLE
Update help.py

### PR DESCRIPTION
**Pull request by EverTokki**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-midnight
```
Comment (abraker95) [inline _config.yml]: The heck is this?

File: sigma/plugins/utility/other/help.py [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
-        help_out = discord.Embed(type='rich', title='❔ Help', color=0x1B6F5F)
-        help_out.set_author(name='Apex Sigma', url=MainServerURL, icon_url='https://i.imgur.com/WQbzk9y.png')
-        help_out.add_field(name='Website', value='[**LINK**](' + MainServerURL + ')')
-        help_out.add_field(name='Commands', value='[**LINK**](' + MainServerURL + 'commands)')
-        help_out.add_field(name='GitHub', value='[**LINK**](https://github.com/aurora-pro/apex-sigma)')
-        help_out.add_field(name='Official Server', value='[**LINK**](https://discordapp.com/invite/Ze9EfTd)')
-        help_out.add_field(name='Add Me', value=f'[**LINK**](https://discordapp.com/oauth2/authorize?client_id={cmd.bot.user.id}&scope=bot&permissions=8)')
-        help_out.set_footer(text=f'Example: {Prefix}help greetmsg', icon_url='https://i.imgur.com/f4TyYMr.png')
-        help_out.set_image(url='https://i.imgur.com/TRSdGni.png')
+        directory = 'sigma/plugins'
```
Comment (abraker95) [inline sigma/plugins/utility/other/help.py]: What does this entire addition do? Add some comments. I know most of the code is uncommented, but we can start making that change now.

File: sigma/plugins/utility/other/help.py [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+                if file == 'plugin.yml':
+                    file_path = (os.path.join(root, file))
+                    with open(file_path) as plugin_file:
+                        plugin_data = yaml.safe_load(plugin_file)
+                        try:
+                            category = plugin_data['categories'][0]
+                            if category.title() not in module_list and category not in ['administration', 'special']:
+                                module_list.append(category.title())
+                        except:
+                            pass
```
Comment (abraker95) [inline sigma/plugins/utility/other/help.py]: What does this entire addition do? Add some comments. I know most of the code is uncommented, but we can start making that change now.

**Review by abraker95 — COMMENTED** [review]
.

File: _config.yml (hunk 1 above)
Comment (EverTokki) [reply _config.yml]: i have no idea sir

File: sigma/plugins/utility/other/help.py (hunk 3 above)
Comment (EverTokki) [reply sigma/plugins/utility/other/help.py]: This was a copy paste of modules.py which can be seen from line 45 [in the deprecated sigma source code.](https://github.com/lu-ci/apex-sigma/blob/42c9e6ad5c66c1e6a8f9c213182df533642727d2/sigma/core/utils.py)

Comment (abraker95) [reply sigma/plugins/utility/other/help.py]: What is it for?

File: _config.yml (hunk 1 above)
Comment (abraker95) [reply _config.yml]: Remove then

File: sigma/plugins/utility/other/help.py [hunk 4]
```diff
@@ -12,24 +12,10 @@
...
-                            category = plugin_data['categories'][0]
-                            if category.title() not in module_list and category not in ['administration', 'special']:
-                                module_list.append(category.title())
-                        except:
-                            pass
 
         help_out = discord.Embed(type='rich', title='❔Help❔', color=0x2cefe5)
         
-        help_out.add_field(name=' Sickle\'s Module List', value='```yaml\n' + '\n'.join(module_list) + '\n```', inline=True)
+        help_out.add_field(name=' Sickle\'s Module List', value='```yaml\n' + '\n'.join(cmd.bot.module_list) + '\n```', inline=True)
```
Comment (EverTokki) [inline sigma/plugins/utility/other/help.py]: Deleted excess code.

**Review by abraker95 — CHANGES_REQUESTED** [review]
Remove the file. Add it to .gitignore. Don't just change it

File: sigma/plugins/utility/other/help.py [hunk 5]
```diff
@@ -1,25 +1,30 @@
-﻿import discord
+import discord
 from config import Prefix, MainServerURL
 from sigma.core.command_alts import load_alternate_command_names
+import os
+import yaml
```
Comment (abraker95) [inline sigma/plugins/utility/other/help.py]: Is yaml being used anywhere?

Comment (abraker95) [reply sigma/plugins/utility/other/help.py]: Same goes for os

Comment (EverTokki) [reply sigma/plugins/utility/other/help.py]: removed

File: .gitignore [hunk 6]
```diff
@@ -31,3 +31,5 @@ border.png
 profile.png
 lolsig.png
 /.vscode/launch.json
+sigma/plugins/nsfw/plugin.yml
```
Comment (abraker95) [inline .gitignore]: Why is this added to .gitignore?

Comment (EverTokki) [reply .gitignore]: I only remember putting in config.yml so probably a mistake. Removed.

**Review by abraker95 — APPROVED** [review]
Good

**Comment by EverTokki** [comment]
god bless
